extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -531,6 +531,18 @@ private JibContainerBuilder createContainerBuilderFromNative(ContainerImageConfi
         }
     }
 
+    private Map<String, String> getEnvironmentVariables(JibConfig jibConfig) {
+        Map<String, String> original = jibConfig.environmentVariables;
+        if (original.isEmpty()) {
+            return original;
+        }
+        Map<String, String> converted = new HashMap<>();
+        for (Map.Entry<String, String> entry : original.entrySet()) {
+            converted.put(entry.getKey().toUpperCase().replace('-', '_').replace('.', '_').replace('/', '_'), entry.getValue());
```

Comment:
Don't we have a utility method for this? I remember seeing this piece of logic somewhere.

Reply:
Not in any available place. 

We should as an improvement introduce a utility class

Reply:
Makes sense.